src/symbol/observable.ts

```diff
@@ -0,0 +1,18 @@
+import {root} from '../util/root';
+
+const Symbol: any = root.Symbol;
+
+export let $$observable: any;
+
+if (typeof Symbol === 'function') {
+  if (!Symbol.observable) {
+    if (typeof Symbol.for === 'function') {
+      $$observable = Symbol.for('observable');
+    } else {
+      $$observable = Symbol('observable');
```

Comment:
I generally agree with @zenparsing that we should just use `Symbol.for()` and not `Symbol()`